src/createServer.js

```diff
@@ -1,3 +1,55 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const query = Object.fromEntries(url.searchParams.entries());
+    const targetCase = query.toCase;
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    res.setHeader('Content-type', 'application/json');
+
+    const errors = [];
+
+    if (!originalText) {
+      errors.push({ message: 'Text to convert is required. '
+      + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    };
+
+    if (!targetCase) {
+      errors.push({ message: '"toCase" query param is required. '
+      + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    } else if (!(cases.includes(targetCase))) {
+      errors.push({ message: 'This case is not supported. '
+      + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.' });
```

Comment:
```suggestion
      + `Available cases: ${cases.join(', ')}.` });
```
You can construct list from array.